lib/src/translations.dart

```diff
@@ -0,0 +1,39 @@
+class Translations {
+  final Map<String, dynamic> _translations;
+  final Map<String, dynamic> _cachedNestedKeys;
+
+  Translations(this._translations) : this._cachedNestedKeys = {};
+  String get(String key) =>
+      (isNestedKey(key) ? getNested(key) : _translations[key]);
+
+  String getNested(String key) {
```

Comment:
What will happen if the localization file is in `simple JSON` format ([such as the one used in react-intl](https://phrase.com/docs/guides/formats/react-intl-simple-json/))? As we break down the key in dots, we cannot access  a key that is not nested, but has dots